lib/dependency-checker.js

```diff
@@ -16,9 +16,12 @@ function isUnsatisfied(pkg) {
 function EmberCLIDependencyChecker(project) {
   this.name    = 'ember-cli-dependency-checker';
   this.project = project;
-  this.checkDependencies();
 }
 
+EmberCLIDependencyChecker.prototype.included = function(/*app*/) {
+  this.checkDependencies();
```

Comment:
the included hook currently gets called twice, do to a quirk that isn't fixed yet. We may want to guard.

cc @rwjblue should confirm